test/admin_tool_test.py

```diff
@@ -58,6 +58,7 @@ def srv(request, tmpdir_factory):
 
     conf = DAEMON_CONF.format(
         run_dir=str(tmp_dir),
+        local_socket=f"{tmp_dir}/sock",
```

Comment:
Looking at the change, I think we should just use a random
address in the abstract namespace, since the local socket is using
the abstract namespace, and we want to keep the test close to the
real usage.

The standard address is:

    "\0/org/ovirt/imageio"

So we can make it random using:

    f"\0/org/ovirt/imageio/{uuid.uuid4()}"